openeo/rest/datacube.py

```diff
@@ -2769,8 +2769,13 @@ def sar_backscatter(
         .. versionchanged:: 0.4.10 replace `orthorectify` and `rtc` arguments with `coefficient`.
         """
         try:
-            schema = Process.from_dict(self.connection.describe_process("sar_backscatter")).get_parameter("coefficient").schema
-            coefficient_options = schema.get_enum_options("enum") + [None]
+            parameter = Process.from_dict(self.connection.describe_process("sar_backscatter")).get_parameter(
+                "coefficient"
+            )
+            schema = parameter.schema
+            coefficient_options = schema.get_enum_options() + [None]
```

Comment:
actually, I think that `get_enum_options` could detect that `None` is also a possible value from https://github.com/Open-EO/openeo-processes/blob/6141771129d58d2a292db3b91d80c812c49f4e52/proposals/sar_backscatter.json#L36-L51